README.md

```diff
@@ -119,6 +119,7 @@ jobs:
           node-version: 12
           registry-url: 'https://registry.npmjs.org'
         if: ${{ steps.release.outputs.release_created }}
+      # if you are using Yarn, substitute the command below with `yarn install --frozen-lockfile`
       - run: npm ci
         if: ${{ steps.release.outputs.release_created }}
       - run: npm publish
```

Comment:
Do folks using yarn still use `npm` for publication?

Reply:
Because I only use Yarn, I do not have a `package.lock` nor a `shrinkwrap` file in my repo and that caused the above to fail with the error mentioned in the issue.